setup.py

```diff
@@ -87,4 +87,17 @@ def is_requirement(line):
     install_requires=load_requirements('requirements/base.in'),
     package_data=package_data("xblockutils", ["public", "templates", "templatetags"]),
     url='https://github.com/edx/xblock-utils',
+    classifiers=[
+        'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.0',
+        'Framework :: Django :: 2.1',
```

Comment:
We shouldn't include 2.0 and 2.1 until the tests are actually passing for those.